### PR TITLE
Mascot updates

### PR DESCRIPTION
**Pull request by wifelette**
Updating the FAQ, commission pricing sheet and request form to reflect addition of corporate mascot option. 

Also deleting/cleaning up an old page from this effort that never went anywhere.

### CODE REVIEW

File: source/tomster/commission.html.erb [hunk 1]
```diff
@@ -65,26 +65,38 @@ title: "Tomster Commission Request"
               <span class="choice">
                 <input id="Field8_0" name="Field8" type="radio" class="field radio" value="Classic Tomster with Simple Shirt Change, $80" tabindex="5" checked="checked" required>
                 <label for="Field8_0">
-                  Classic Tomster or Zoey with Simple Shirt Change, $80
+                  UG Classic Tomster or Zoey with Simple Shirt Change, $80
```
Comment (acorncom) [inline source/tomster/commission.html.erb]: UG standing for user group I'm assuming?  Think it'd be worth spelling that out so it's clearer on this form?

Comment (wifelette) [inline source/tomster/commission.html.erb]: Ack, just an oversight, good catch!

Comment (wifelette) [inline source/tomster/commission.html.erb]: I was just trying to keep the line breaks tidy :p

Comment (acorncom) [inline source/tomster/commission.html.erb]: That's what I figured 😀 I vote for non-tidy 😛

Comment (locks) [inline source/tomster/commission.html.erb]: It doesn't look good ;X Can we explain beforehand and use UG and CO?

Comment (wifelette) [inline source/tomster/commission.html.erb]: @locks That works for me!

**Pull request merged**